pipeline/notifications/render_check_message.py

```diff
@@ -0,0 +1,28 @@
+from pipeline.notifications.renderer import RenderNotification
+from pipeline.types import NotificationInfo
+
+
+class NewMessageRender(RenderNotification):
+    def __init__(self):
+        pass
+
+    def render(self, inp: NotificationInfo) -> str:
+
+        # if the notification type doesn't match the type of render, return immediately saying as much
+        if inp.notification_type.value != "check_message":
```

Comment:
You can compare the enum here. Also, raise an error instead of returning a string, otherwise the caller might consider the string to be a valid message!

```python
# You can move this exception to a file like `pipeline/notification/exceptions.py`
# or put it in `pipeline/notifications/renderer.py` since it will be used by other renderers
class InvalidNotificationTypeException(Exception):
  pass

if inp.notification_type != NotificationType.CHECK_MESSAGE:
  raise InvalidNotificationTypeException("This renderer only supports CHECK_MESSAGE notifications.")
```